README.md

```diff
@@ -22,17 +22,23 @@ At a very high level, the code review process should go as follows:
 - Reviewers review the code and leave feedback for the author
 - Once a reviewer has marked the PR approved, the author can merge the code.
 
-In general, the onus is on the author to convince the reviewer that the changes are acceptable. However, the onus is on the reviewer to ensure that the changes requested are reasonable. 
+In general, the onus is on the author to convince the reviewer that the changes are acceptable. However, the onus is on the reviewer to ensure that the changes requested are reasonable.
 
 In extremely rare circumstances where the author and reviewer cannot reach consensus, the review can be escalated, however this should be done in only the most extreme circumstances and should never be necessary if both the author and the reviewer are following the advice below.
 
+Aim to review PRs within one business day, but use your judgement as to when to break this guideline. Common considerations:
```

Comment:
The 1-business-day guideline is about your first review of a PR after it's submitted, right? And I think independently also then applies to reviewing the changes someone made in response to your comments?

I.e. we don't expect all of the review, back and forth, and approval to happen in 24 hours except in the simplest of cases (where there reviewer approves without blocking comments). Is that right?

Reply:
Good point, will clarify. My understanding is the same as yours.